README.adoc

```diff
@@ -145,21 +156,23 @@ openlibertydumps          oldump,oldumps     openliberty.io   true         OpenL
 openlibertytraces         oltrace,oltraces   openliberty.io   true         OpenLibertyTrace
 ```
 
-The Open Liberty Operator introduces new CRDs,including the `OpenLibertyApplication` CRD. 
+CRDs define a `kind` of object that can be used. 
+The Open Liberty Operator uses the `OpenLibertyApplication` `kind`. 
 The `OpenLibertyApplication` CRD defines a set of configurations for
 deploying an Open Liberty-based application, including the application image, number of instances, and storage settings.
-The Open Liberty Operator watches for changes to instances of the OpenLibertyApplication CRD and creates Kubernetes resources that are based on the configuration that is defined in the CRD instance.
+The Open Liberty Operator watches for changes to instances of the `OpenLibertyApplication` object `kind` and creates Kubernetes resources that are based on the configuration that is defined in the CRD.
 
-You can use the `OpenLibertyApplication` resource in the upcoming sections.
+You will be using the `OpenLibertyApplication` object `kind` in the upcoming sections.
+It can also be used by using any of its `shortnames` such as `olapps`.
 
-Run the following command to view all the supported API resources that are available through the Strimzi Operator:
+Run the following command to view all CRDs used by the Strimzi Operator:
 
 [role='command']
 ```
 oc api-resources --api-group=kafka.strimzi.io
 ```
 
-Look for the following output, which shows the CRDs that are installed by the Strimzi Operator:
+Look for the following output, which shows the CRDs that are used by the Strimzi Operator:
```

Comment:
suggested edit

Look for the following output, which lists the CRDs that the Strimzi Operator uses, along with object kinds and shortnames.